packages/react-ui-core/src/DatePicker/DatePicker.js

```diff
@@ -136,15 +137,16 @@ export default class DatePicker extends PureComponent {
           onBlur={this.validateTextField}
         />
 
-        {this.state.showCalendar &&
+        {this.state.showCalendar && (
```

Comment:
add to destructuring assignment on line 128?

Reply:
Yep, I can do that. 👍

Reply:
Done!